exercises/blink_blink/exercise.js

```diff
@@ -36,22 +32,15 @@ exercise.addVerifyProcessor(verifyProcessor(exercise, function (test, done) {
   var led = five.Led.instances[0]
 
   test.truthy(led, 'create_led_instance')
-  test.equals(led.pin, pins.led, 'connect_led_to_pin', {pin: pins.pins})
-  test.truthy(led.strobe.called || led.blink.called, 'led_flashing')
+  test.truthy(led.pin, 'connect_led_to_pin', { pin: led.pin })
+  test.truthy(led.strobe.called || led.blink.called, 'blink_strobe_called')
 
   if (led.blink.called) {
     test.equals(led.blink.getCall(0).args[0], 1000, 'led_flashing')
   } else {
     test.equals(led.strobe.getCall(0).args[0], 1000, 'led_flashing')
   }
 
-  // should have set pin 13 into digital output mode
-  test.truthy(io.pinMode.calledWith(13, io.MODES.OUTPUT), 'pin_mode', {pin: 2, mode: 'OUTPUT'})
```

Comment:
Mode setting is wrapped in johnny five `Led` constructor. It's not part of the solution.